assets/src/edit-story/components/panels/textStyle/padding/padding.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { useCallback } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Label, Row, MULTIPLE_VALUE } from '../../../form';
+import { useCommonObjectValue } from '../../utils';
+import LockedPaddingControls from './locked';
+import UnlockedPaddingControls from './unlocked';
+
+export const DEFAULT_PADDING = { horizontal: 0, vertical: 0, locked: true };
+
+function PaddingControls({ selectedElements, pushUpdateForObject }) {
+  const padding = useCommonObjectValue(
+    selectedElements,
+    'padding',
+    DEFAULT_PADDING
+  );
+
+  // When multiple element selected with padding locked value combined, it treated as false, reversed behavior with aspect lock ratio.
+  const lockPadding =
+    padding.locked === MULTIPLE_VALUE ? false : padding.locked;
+
+  const handleChange = useCallback(
+    (newPadding, submit = false) => {
```

Comment:
Looks like some test assertions need to be updated now that `false` is passed by default (vs. nothing before)